app/views/property_searches/_form.html.erb

```diff
@@ -1,34 +1,16 @@
-<div class="accordion mb-4">
-  <div class="card u-card-collapse border-0">
-    <div class="card-header px-4 bg-white border-0" id="filters-heading">
-      <div class="collapsed" data-toggle="collapse" data-target="#filters" aria-expanded="false" aria-controls="filters" role="button">
-        <h3 class="h6 d-flex justify-content-between align-items-center font-weight-normal u-card-collapse__heading mb-0">
-          <span><i class="fas fa-search"></i></span>
-          <span class="fa fa-angle-up u-card-collapse__heading-icon"></span>
-        </h3>
-      </div>
-    </div>
-    <div id="filters" class="collapse <%= "show" if @property_search.present? %>" aria-labelledby="filters-heading" data-parent="#filters-heading">
-      <div class="card-body px-4">
-        <%= simple_form_for @property_search do |f| %>
-          <div class="row mt-5">
-            <%= tag.div f.input(:name, required: false), class: "col-12 col-md-auto" %>
-            <%= tag.div f.input(:user, label: "Publisher", required: false), class: "col-12 col-md-auto" %>
-            <%= tag.div f.input(:property_types, collection: property_types_for_select, required: false, input_html: { multiple: true, data: { controller: "select" }}), class: "col-12 col-md-auto" %>
-            <%= tag.div f.input(:statuses, collection: property_statuses_for_select, required: false, input_html: { multiple: true, data: { controller: "select" }}), class: "col-12 col-md-auto" %>
-            <%= tag.div f.input(:ad_templates, collection: ad_templates_for_select, required: false, input_html: { multiple: true, data: { controller: "select" }}), class: "col-12 col-md-auto" %>
-            <%= tag.div f.input(:languages, collection: languages_for_select, required: false, input_html: { multiple: true, data: { controller: "select" }}), class: "col-12 col-md-auto" %>
-            <%= tag.div f.input(:keywords, collection: keywords_for_select, label: "Topics", required: false, input_html: { multiple: true, data: { controller: "select" }}), class: "col-12 col-md-auto" %>
+<%= simple_form_for @property_search, wrapper: :filter_form do |f| %>
+    <%= f.input(:name, required: false) %>
+    
+    <%# TODO: Is this something we should cache or have elsewhere? %>
+    <%= f.input(:user_id, label: "Publisher", collection: User.publisher.order(last_name: :asc).map { |u| [u.name, u.id] }, required: false, input_html: { data: { controller: "select" }}) %>
+    <%# ENDTODO %>
```

Comment:
We'll probably want to cache this. but we can punt for now I think. We may also want to restrict it to `administrators` only.